index-webpack.scss

```diff
@@ -0,0 +1,3 @@
+$use-embedded-font: true;
+
+@import 'index';
```

Comment:
Why do we need this file?

Reply:
this is added to switch the fonts from a url to being embedded (the default true is set on top of the icons file)

Reply:
It looks like a change which is actually not necessary to this package but to your setup^^